### PR TITLE
Use Tharsis all-in-one-Docker for acceptance tests

### PR DESCRIPTION
**Pull request by amanshallow**

### CODE REVIEW

File: internal/provider/resource_group.go [hunk 1]
```diff
@@ -229,7 +229,8 @@ func (t *groupResource) Delete(ctx context.Context,
 	// Delete the group via Tharsis.
 	err := t.client.Group.DeleteGroup(ctx,
 		&ttypes.DeleteGroupInput{
-			ID: ptr.String(state.ID.ValueString()),
+			ID:    ptr.String(state.ID.ValueString()),
+			Force: ptr.Bool(true), // FIXME: this will always be true.
```
Comment (bbdev442) [inline internal/provider/resource_group.go]: This force can be removed.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -45,34 +45,52 @@ jobs:
...
         # list whatever Terraform versions here you would like to support
         terraform:
-          - '1.0.*'
-          - '1.1.*'
-          - '1.2.*'
-          - '1.3.*'
+          - "1.0.*"
+          - "1.1.*"
+          - "1.2.*"
+          - "1.3.*"
```
Comment (bbdev442) [inline .github/workflows/test.yml:75]: We might as well add the newer cli versions to this list while it's being updated.

Comment (amanshallow) [reply .github/workflows/test.yml:75]: The latest CLI version is [1.37](https://github.com/hashicorp/terraform/releases). Won't the wildcard at the end cover it?

Comment (bbdev442) [reply .github/workflows/test.yml:75]: That's my mistake, I misread that version.

**Review by bbdev442 — APPROVED** [review]

File: go.sum [hunk 3]
```diff
@@ -323,6 +323,8 @@ github.com/zclconf/go-cty v1.12.1/go.mod h1:s9IfD1LK5ccNMSWCVFCE2rJfHiZgi7JijgeW
 github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b/go.mod h1:ZRKQfBXbGkpdV6QMzT3rU1kSTAnfu1dO8dPKjYprgj8=
 gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-sdk-go v0.9.0 h1:vs9YG89YIj3yu3eaSjipxifOAIRcSpA3PcUWzbV7DeM=
 gitlab.com/infor-cloud/martian-cloud/tharsis/tharsis-sdk-go v0.9.0/go.mod h1:Mo8oWON5Bq1053r1pA+G0CHfTUAEofv2Mq/JlHTFOiE=
```
Comment (rmrichesjr1) [inline go.sum:325]: go.sum still shows Tharsis SDK v0.9.0 (as well as 0.10.0).  Was "go mod tidy" run?

Comment (amanshallow) [reply go.sum]: Nice, catch. Forgot about that. Thanks.

**Review by rmrichesjr1 — APPROVED** [review]

**Review by bbdev442 — APPROVED** [review]

**Pull request merged**